.github/CODEOWNERS

```diff
@@ -1,3 +1,2 @@
 # Snyk Code will be required for a review on every PR
 
```

Comment:
Let's not abandon this repo and leave it without owner.
Should be @snyk/zenith.

Reply:
https://snyk.slack.com/archives/C0370TRPF7F/p1649346761713119?thread_ts=1649321913.624999&cid=C0370TRPF7F